dms/tests/test_file.py

```diff
@@ -84,3 +90,9 @@ def test_content_file_res_mimetype(self):
         self.assertEqual(file_svg.res_mimetype, "image/svg+xml")
         file_logo = self.env.ref("dms.file_02_demo")
         self.assertEqual(file_logo.res_mimetype, "image/jpeg")
+
+    def test_content_file_res_mimetype_magic_library(self):
+        if not magic:
+            self.skipTest("Without python-magic library installed")
```

Comment:
FTR there's a decorator for this: https://docs.python.org/3/library/unittest.html#unittest.skipIf

However it's OK as it is 😊